build/transpile.js

```diff
@@ -2430,9 +2431,7 @@ class Transpiler {
             pyAsync: [
                 "import asyncio",
                 "import ccxt.async_support as ccxt  # noqa: E402",
-                "",
-                "",
-                "",
+                ""
```

Comment:
all the changes inside `transpile.js` is done within `transpileExamples` method, so no other areas are affected.

Reply:
@ttodua is it possible to move this logic out of this PR? and add it in a different one? or is it required for the proxies' examples?

Reply:
the changes in this method were solely related to the examples transpilation yes, because it was first time when I added code related to WS things, and it turned out some changes were needed. 
exactly here where you commented, if you mean just empty lines addition -  the empty lines are not that much critical, just for readability it was better